File: CodeChef/Beginner/05FindRemainder/findremainder.cpp
#include <iostream>
using namespace std;
class findremainder {};

int main() {
  int n, a, b;
  cin >> n;
  for (int i = 0; i < n; i++) {
    cin >> a;
    cin >> b;
    cout << a % b << endl;
  }
}

File: CodeChef/Beginner/05FindRemainder/findremainder.java
import java.util.Scanner;

class findremainder {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, a, b;
        n = sc.nextInt();
        for (int i = 0; i < n; i++) {
            a = sc.nextInt();
            b = sc.nextInt();

            System.out.println(a % b);
        }
        sc.close();
    }
}
